pkg/apis/serving/v1alpha2/explainer.go

```diff
@@ -20,8 +20,7 @@ import (
 	"k8s.io/klog"
 )
 
-// +k8s:openapi-gen=false
-// +k8s:deepcopy-gen=false
+// +kubebuilder:object:generate=false
```

Comment:
We should consider moving these interfaces into pkg/controller or somewhere else. The interfaces can be implemented by our types, but defined in a separate package to keep the generation simple.